src/CLI/clitree/cli-xml/image.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE CLISH_MODULE [
+]>
+<CLISH_MODULE
+      xmlns="http://www.dellemc.com/sonic/XMLSchema"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema 
+      http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
+      >
+  <VIEW name="enable-view">
+    <COMMAND
+         name="show image-list"
+         help="Show list of images"
+         >
+         <ACTION>
+              python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components show_image.j2 ${__full_line}
+         </ACTION>
+    </COMMAND>
+
+    <COMMAND
+         name="image"
+         help="Image related commands"
+         >
+         <ACTION>
```

Comment:
Do we have a command #image?

Reply:
We don't need. Do you want me to put a no op there ?